src/test/java/io/epirus/console/account/AccountManagerTest.java

```diff
@@ -29,37 +35,71 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class AccountManagerTest {
+    @TempDir static File workingDirectory;
     private static final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
-    private static final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
     private static final PrintStream originalOut = System.out;
-    private static final PrintStream originalErr = System.err;
+    static OkHttpClient mockedOkHttpClient = mock(OkHttpClient.class);
+    static Call call = mock(Call.class);
+    static ConnectionPool connectionPool = mock(ConnectionPool.class);
+    static AccountManager accountManager;
+    static CliConfig cliConfig;
 
     @BeforeAll
-    public static void setUpStreams() {
+    public static void setUp() throws IOException {
+        cliConfig =
+                new CliConfig(
+                        "4.6.0-SNAPSHOT",
+                        "https://auth.epirus.io",
+                        "ce3724d9-06eb-4b50-b654-ae0e6a756caf",
```

Comment:
Is this token actually active, or for mocking? Not sure we should leave tokens in source code, but probably fine if for mocking only.

Reply:
This token can be replaced with any value it should not fail the tests. In a config file without an account, the token would be null.